crypto/x509v3/v3_addr.c

```diff
@@ -1180,9 +1184,13 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     int i, j, ret = 1;
     X509 *x;
 
-    OPENSSL_assert(chain != NULL && sk_X509_num(chain) > 0);
-    OPENSSL_assert(ctx != NULL || ext != NULL);
-    OPENSSL_assert(ctx == NULL || ctx->verify_cb != NULL);
+    if (!ossl_assert(chain != NULL && sk_X509_num(chain) > 0)
```

Comment:
You said "this adds value" and I said "it appears that only first kind of does". But then there is kind of equivalent check few lines below, i.e. beyond these particular three lines. So that implied question is why two checks?